hq/app/aws/iam/IAMClient.scala

```diff
@@ -46,10 +61,9 @@ object IAMClient {
         _ <- Retry.until(generateCredentialsReport(client), CredentialsReport.isComplete, "Failed to generate credentials report", delay)
         report <- getCredentialsReport(client)
         stacks <- CloudFormation.getStacksFromAllRegions(account, cfnClients, regions)
-        enrichedReport = CredentialsReport.enrichReportWithStackDetails(report, stacks)
-      } yield {
-        CredentialsReportDisplay.toCredentialReportDisplay(enrichedReport)
-      }
+//        reportWithTags <- getCredentialTags(report, client) NOTE: disabled pending fixing the stack set
```

Comment:
Due to problems with the stack set, the feature is disabled for now so that this PR can be merged (to reduce merge conflicts later)